week-1/K-functions-parameters/exercise5.js

```diff
@@ -1,5 +1,7 @@
 // Declare your function here
-
+function createLongGreeting(name, b) {
```

Comment:
can you change the second parameter to be meaningful (it should describe the variable)